### PR TITLE
stonewalling

### PR DESCRIPTION
**Pull request by queclub**
@
[New Compressed (zipped) Folder.zip](https://github.com/ankit/stylebot/files/6706059/New.Compressed.zipped.Folder.zip)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@
 [![Follow on Twitter](https://badgen.net/twitter/follow/ahujaankit)](https://twitter.com/ahujaankit)
 ![License](https://img.shields.io/github/license/ankit/stylebot)
 
-Stylebot is a browser extension that lets you change the appearance of the web instantly. Available on [Firefox](https://addons.mozilla.org/en-US/firefox/addon/stylebot-web/) and [Chrome](https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha).
+Stylebot is a browser extension that lets you change the appearance of the web instantly.
+Available on [Chrome](https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha), [Firefox](https://addons.mozilla.org/en-US/firefox/addon/stylebot-web/) and [Edge](https://microsoftedge.microsoft.com/addons/detail/stylebot/mjolbpfednnbebfapicajpifliopnnai).
 
 - Easy to use: Pick and style elements using UI actions
 - Simple & Quick: Changes are saved instantly
```
Comment (queclub) [inline README.md:16]: {"version":"2.0.104.12","settings":{"blur":0,"brightness":100,"contrast":100,"grayscale":0,"huerotate":0,"invert":0,"saturate":100,"sepia":0,"applyvideofilters":false,"backgroundcolor":"#000000","backgroundopacity":50,"blackbars":false,"blockads":false,"blockadsexceptforsubs":true,"blockautoplay":true,"blockhfrformats":false,"blockwebmformats":false,"cinemamode":false,"cinemamodewideplayer":false,"controlbar":{"active":true,"autohide":false,"centered":false,"position":"absolute"},"controls":["loop","stop","reverse-playlist","volume-booster","whitelist","not-interested","cards-end-screens","cinema-mode","size","pop-up-player","speed-minus","speed","speed-plus","video-filters","flip-horizontally","flip-vertically","screenshot","keyboard-shortcuts","custom-script","options"],"controlsvisible":false,"controlspeed":false,"controlspeedmousebutton":false,"controlvolume":true,"controlvolumemousebutton":false,"customcolors":{"--main-color":"#00adee","--main-background":"#111111","--second-background":"#181818","--hover-background":"#232323","--main-text":"#eff0f1","--dimmer-text":"#cccccc","--shadow":"#000000"},"customcssrules":"","customscript":"","customtheme":false,"darktheme":true,"date":1624501756028,"defaultvolume":false,"disableautoplay":true,"executescript":false,"expanddescription":false,"filter":"none","hidecardsendscreens":false,"hidechat":false,"hidecomments":false,"hiderelated":false,"ignoreplaylists":false,"ignorepopupplayer":false,"localecode":"en_US","localedir":"ltr","message":false,"miniplayer":false,"miniplayerposition":"_top-left","miniplayersize":"_400x225","newestcomments":false,"overridespeeds":false,"pauseforegroundtab":false,"pausevideos":false,"popuplayersize":"640x360","qualityembeds":"medium","qualityembedsfullscreen":"hd1080","qualityplaylists":"hd720","qualityplaylistsfullscreen":"hd1080","qualityvideos":"hd720","qualityvideosfullscreen":"hd1080","reload":false,"reversemousewheeldirection":false,"selectquality":false,"selectqualityfullscreenoff":true,"selectqualityfullscreenon":false,"speed":1,"speedvariation":0.05,"stopvideos":false,"theatermode":false,"theme":"default-dark","themevariant":"youtube-deep-dark-9anime.css","update":0,"volume":50,"volumemultiplier":10,"volumevariation":5,"whitelist":"","wideplayer":false,"wideplayerviewport":false}}

**Review by queclub — COMMENTED** [review]
System Memory Usage  |  Download 
Monitor system's memory via toolbar popup or toolbar button's icon.

Some of the important features of the System Memory Usage addon are listed below. This is followed by an overview of this addon. This summary contains few words about the addon, its function, and its features. Next is FAQ, this section is useful to get information about how this addon works in your browser. If you couldn’t find a specific answer, please fill the bug report form on this page or the contact form on the homepage to let us know about your question. Following the FAQ, is the bug report form. It is designed to collect user feedbacks and bugs about this addon. Users can suggest improvements or feature requests through this form too. The next section is for submitting comments about this addon which uses the Facebook comments plugin. The last section is dedicated to reports submitted through the GitHub website. We hope the information provided on this page, could help you get all the basics you need about this addon and enable you to have a better experience with System Memory Usage.

In short, System Memory Usage is a browser extension that inform you about the amount of memory your system is currently using.
4 Reasons to Use System Memory Usage

**Comment by queclub** [comment]
System Memory Usage  |  Download 
Monitor system's memory via toolbar popup or toolbar button's icon.

Some of the important features of the System Memory Usage addon are listed below. This is followed by an overview of this addon. This summary contains few words about the addon, its function, and its features. Next is FAQ, this section is useful to get information about how this addon works in your browser. If you couldn’t find a specific answer, please fill the bug report form on this page or the contact form on the homepage to let us know about your question. Following the FAQ, is the bug report form. It is designed to collect user feedbacks and bugs about this addon. Users can suggest improvements or feature requests through this form too. The next section is for submitting comments about this addon which uses the Facebook comments plugin. The last section is dedicated to reports submitted through the GitHub website. We hope the information provided on this page, could help you get all the basics you need about this addon and enable you to have a better experience with System Memory Usage.

In short, System Memory Usage is a browser extension that inform you about the amount of memory your system is currently using.
4 Reasons to Use System Memory Usage

**Comment by queclub** [comment]
https://youtu.be/qw-qR8qZC5A

**Comment by queclub** [comment]
blob:https://www.youtube.com/ce063fb8-a64c-4186-9c97-e43ba49b13d8
